src/XCCDF/public/xccdf_session.h

```diff
@@ -42,6 +42,19 @@
 struct xccdf_session;
 
 /**
+ * Loading flags for XCCDF session
+ * @memberof xccdf_session
+ * The flags set which components will be loaded in XCCDF session.
+ */
+typedef enum {
+	XCCDF_SESSION_LOAD_NONE = 0,
```

Comment:
> The problem is that since it's zero there is no way to "remove" it from the bitfield. 

Oh, now I see it.

> Tbh I can't find a meaningful example of creating an XCCDF session without loading an XCCDF document 😬

I understand you, but who knows what use cases might come up in the future? 😬
And I see XCCDF_SESSION_LOAD_XCCDF to be non zero and removable from bitfield as  future proof implementation.

So, I'd suggest to have
XCCDF_SESSION_LOAD_NONE = 0 and 
XCCDF_SESSION_LOAD_XCCDF = 1 << 0
and shift the other values.